ros_ws/src/teleoperation/include/joystick_map_ps3.h

```diff
@@ -0,0 +1,33 @@
+#include "joystick_map.h"
+
+class JoystickMapPs3 : public JoystickMap
+{
+    public:
+    JoystickMapPs3()
+    {
+    }
+
+    ~JoystickMapPs3()
+    {
+    }
+
+    float getSteeringAxis(const sensor_msgs::Joy::ConstPtr& joystick)
+    {
```

Comment:
**MAY**
If you implement a member function in a header file you may want to add the `inline` keyword as otherwise the same member function might be implemented twice by different source files (of both include this header file) which leads to link time errors.